MekHQ/src/mekhq/campaign/parts/MissingEnginePart.java

```diff
@@ -267,17 +266,16 @@ public boolean isInLocation(String loc) {
              case Engine.XXL_ENGINE:
                  needsSideTorso = true;
                  break;
+             default:
+                 break;
          }
-         if (needsSideTorso
-                 && (unit.getEntity().getLocationFromAbbr(loc) == Mech.LOC_LT
-                         || unit.getEntity().getLocationFromAbbr(loc) == Mech.LOC_RT)) {
-             return true;
-         }
-         return false;
+        return needsSideTorso
+                && ((unit.getEntity().getLocationFromAbbr(loc) == Mech.LOC_LT)
+                || (unit.getEntity().getLocationFromAbbr(loc) == Mech.LOC_RT));
```

Comment:
Future note: these sorts of changes have been making it difficult to keep some of my branches up to date. Lots of merge conflicts unrelated to changes.

Reply:
Parts have been a place that every time I load it I get incredibly annoyed by. This PR clears out a ton of my issues with them.
Sorry that it is causing annoying merge conflicts, but... I had thought you had moved away from working on parts and thus that this would be a good time to finally do the cleanup I had wanted to do.